proto/accessgraph/v1alpha/access_graph_service.proto

```diff
@@ -146,3 +152,25 @@ message ReplaceCAsRequest {
 
 // ReplaceCAsResponse is the response for ReplaceCAs.
 message ReplaceCAsResponse {}
+
+// AWSEventsStreamRequest is a request to send commands to the AWS importer.
+// This command is used to sync the access graph with the AWS database state.
+message AWSEventsStreamRequest {
+  // operation contains the desired operation
+  oneof operation {
+    // sync is a command to sync the access graph with the Teleport database state.
+    // it's issued once Teleport finishes syncing all resources with the database.
+    AWSSyncOperation sync = 1;
+    // upsert is a command to put a resource into the access graph or update it.
+    AWSResourceList upsert = 2;
+    // delete is a command to delete a resource from the access graph when it's deleted from Teleport.
+    AWSResourceList delete = 3;
```

Comment:
Having to transfer the full definition of a resource just to delete it feels somewhat wasteful, but on the bright side it does reduce the number of types to maintain, and possibly isn't that bad in the first place.

Reply:
For now I am just trying to reduce the number of resources to maintain but I do agree with you